code/go/0chain.net/smartcontract/minersc/handler.go

```diff
@@ -796,6 +796,10 @@ func (mrh *MinerRestHandler) getUserPools(w http.ResponseWriter, r *http.Request
 	ups := new(userPools)
 	ups.Pools = make(map[string][]*delegatePoolStat, len(minerPools)+len(sharderPools))
 	for _, pool := range minerPools {
+		if pool.Status >= spenum.PoolStatus(pool.Status).Size() {
```

Comment:
We don't need to do this size checking as the `String()` method will do it for us and return unknown status if it's invalid. We will return it along with other valid info to users